src/core/core.js

```diff
@@ -236,6 +236,7 @@ export default class Core {
         offset: this.storage.get(StorageKeys.SEARCH_OFFSET) || 0,
         skipSpellCheck: this.storage.get(StorageKeys.SKIP_SPELL_CHECK),
         queryTrigger: queryTriggerForApi,
+        sessionId: this.getSessionId(),
```

Comment:
Is sessionId supposed to be send even if sessionTrackingEnabled is false?

Reply:
If `sessionTrackingEnabled` is false, we should not send the `sessionId` in the request. Chris, if the value is false, will `getSessionId()` return `null`? If it is `null` and we include it here, is it filtered out by Core and not included in the resulting request?

Reply:
getSessionId() returns null if sessionTrackingEnabled is false, which I think is the same as when we don't pass in sessionId.